vid2persona/gen/tgi_openllm.py

```diff
@@ -0,0 +1,114 @@
+import json
```

Comment:
I don't know what this is doing.

Reply:
Just ignore it for now. I was about to test the pipeline with TGI, but didn't have enough time